vllm/worker/worker.py

```diff
@@ -233,24 +222,25 @@ def determine_num_available_blocks(self) -> Tuple[int, int]:
         num_gpu_blocks = max(num_gpu_blocks, 0)
         num_cpu_blocks = max(num_cpu_blocks, 0)
 
-        end_time = time.time()
-        logger.info(
-            "Memory profiling results: "
-            "duration=%.2f seconds, "
-            "total_gpu_memory=%.2fGiB, "
-            "initial_memory_usage=%.2fGiB, "
-            "peak_torch_memory=%.2fGiB, "
-            "memory_usage_post_profile=%.2fGiB, "
-            "non_torch_memory=%.2fGiB, "
-            "kv_cache_size=%.2fGiB, "
-            "gpu_memory_utilization=%.2f.", end_time - start_time,
-            total_gpu_memory / (1024**3),
-            (total_gpu_memory - free_memory_pre_profile) / (1024**3),
-            (peak_memory - non_torch_allocations) / (1024**3),
-            total_allocated_bytes / (1024**3),
-            non_torch_allocations / (1024**3),
-            available_kv_cache_memory / (1024**3),
-            self.cache_config.gpu_memory_utilization)
+        msg = ("Memory profiling results:\n"
+               "duration\t"
+               f"{result.profile_time:.2f} seconds\n"
+               "total_gpu_memory\t"
+               f"{(total_gpu_memory / GiB_bytes):.2f}GiB\n"
+               "gpu_memory_utilization\t"
+               f"{self.cache_config.gpu_memory_utilization:.2f}\n"
+               "the current vLLM instance can use\t"
+               f"{(memory_for_current_instance / GiB_bytes):.2f}GiB\n"
+               "model weights take\t"
+               f"{(result.weights_memory_in_bytes / GiB_bytes):.2f}GiB\n"
+               "non_torch_memory\t"
+               f"{(result.non_torch_increase_in_bytes / GiB_bytes):.2f}GiB\n"
+               "PyTorch activation peak memory\t"
+               f"{(result.torch_peak_increase_in_bytes / GiB_bytes):.2f}GiB\n"
+               "available_kv_cache_memory\t"
+               f"{(available_kv_cache_memory / GiB_bytes):.2f}GiB\n")
```

Comment:
Some suggestions on making this a bit nicer:
1. The log would be much easier to parse if the numbers were aligned on the same column, right now they're all over the place
2. The descriptions are a mix of plain words and variable names, for logs maybe we should just use words. `available_kv_cache_memory:` -> `KV Cache Size:` etc.
3. "Non torch memory" is the one item on this list that I think might not be easily understood by somebody reading the logs. Maybe calling it something a little more generic like "Memory overhead" would be less distracting

Reply:
I think all of the newlines take too much space in the logs. IMO it would be more simple to keep the same single-line comma-separated result as before

Reply:
addressed in https://github.com/vllm-project/vllm/pull/10511#issuecomment-2546324136 , PTAL